### PR TITLE
Add Premium Onboarding

### PR DESCRIPTION
**Pull request by maxxcrawford**
Fixes MPP-1087

Related L10N String PR: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/14

TO-DO: 
- [x] Add l10n Strings
- [x] Style UI for onboarding dismissal 
- [x] Add logic to only show to premium users
- [x] Add JS/dismissal functionality
- [x] Edgecase: Premium account with ZERO aliases (double onboarding) 

## Testing steps: 

Test 1: Premium Visible/Dismissal
- Log in with premium account
- **Expected:** Premium onboarding is visible on dashboard page
- Click on dismiss text on bottom of premium onboarding
- **Expected:** Premium onboarding is no longer visible
- Close browser/window and open dashboard page again or refresh tab
- **Expected:** Premium onboarding is not visible

_Edgecase: If a user is premium and has NO aliases created, the premium onboarding should not be visible. (The standard onboarding should be). After creating at least one, the premium onboard is visibel (and the inital onboarding is not)._

Test 2: Non-premium 
- Log in with non-premium account
- **Expected:** Premium onboard is not visible

### CODE REVIEW

**Comment by maxxcrawford** [comment]
@codemist @Vinnl Tagging you both for a review even though this is in draft mode. The only thing remaining is locking in strings. :)

File: static/scss/partials/onboarding.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // Protocol
 @import "libs/protocol/css/templates/multi-column.scss";
+@import "libs/protocol/css/components/zap.scss";
```
Comment (maxxcrawford) [inline static/scss/partials/onboarding.scss:3]: Adding in [zap component](https://protocol.mozilla.org/patterns/atoms/zap.html) from Protocol.

File: static/scss/partials/onboarding.scss [hunk 2]
```diff
@@ -65,3 +66,82 @@
...
+
+    .mzp-u-list-styled li {
+        margin-bottom: $spacing-md;
+    }
+
+    li::marker {
+        color: $color-yellow-50;
+    }
+
+    button {
```
Comment (maxxcrawford) [inline static/scss/partials/onboarding.scss:134]: Not sure if this should just be an `a` link. Depneds on Eduardo's designs TBH.

Comment (maxxcrawford) [reply static/scss/partials/onboarding.scss:154]: Revised to a X in the top right corner.

File: static/scss/partials/onboarding.scss (hunk 1 above)
Comment (maxxcrawford) [reply static/scss/partials/onboarding.scss:3]: _(That's also all the zap.svgs)_

**Review by codemist — CHANGES_REQUESTED** [review]
Looks good, however when I change to a mobile view the exit button stops working.

**Comment by maxxcrawford** [comment]
> Looks good, however when I change to a mobile view the exit button stops working.

Great catch. It was a z-index issue.

**Review by codemist — APPROVED** [review]

File: privaterelay/templates/profile.html [hunk 3]
```diff
@@ -32,7 +32,7 @@
   <div class="messages-profile">
     {% include "includes/messages.html" %}
   </div>
-  {% if user_profile.has_premium %}
+  {% if user_profile.has_premium  %}
```
Comment (groovecoder) [inline privaterelay/templates/profile.html:35]:
```suggestion
  {% if user_profile.has_premium %}
```

**Review by groovecoder — CHANGES_REQUESTED** [review]
obligatory 1-character change request so you know I actually looked at EVERY SINGLE CHARACTER!

File: privaterelay/templates/profile.html (hunk 3 above)
Comment (maxxcrawford) [reply privaterelay/templates/profile.html:35]: :zap:

**Review by groovecoder — APPROVED** [review]

**Pull request merged**